packages/js/components/src/split-dropdown/test/index.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { createElement } from '@wordpress/element';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { SplitDropdown } from '..';
+
+describe( 'SplitDropdown', () => {
+	test( 'it renders correctly collapsed', () => {
+		const { container, getByText, queryByText } = render(
+			<SplitDropdown>
+				<Button>Default Action</Button>
+				<Button className="test-collapsed-button">
+					Secondary Action
+				</Button>
+				<Button className="test-collapsed-button">
+					Tertiary Action
+				</Button>
+			</SplitDropdown>
+		);
+		expect( container ).toMatchSnapshot();
+
+		// should have correct content
+		expect( getByText( 'Default Action' ) ).toBeInTheDocument();
+		expect( queryByText( 'Secondary Action' ) ).not.toBeInTheDocument();
+
+		// should have correct classes
+		const presentClassNames = [
```

Comment:
When possible, I think we should favor using the `getByX` methods built into jest as class names can be an implementation detail.

Checking to see if the dropdown opens when clicked and that those other two items exist would be the most ideal scenario.

Reply:
I've added a click test now and stopped querying by className.